### PR TITLE
GitHub Action: test on current version of Python

### PR DESCRIPTION
**Pull request by cclauss**
Fixes # .

Changes proposed in this pull request:

 -
 -
 -

### CODE REVIEW

**Comment by RobbeSneyders** [comment]
Thx @cclauss. You'll need to create a 3.9 environment in `tox.ini` as well.

File: tox.ini [hunk 1]
```diff
@@ -15,8 +16,9 @@ envlist =
 [gh-actions]
 python =
     3.6: py36-min,py36-pypi
-    3.7: py37-min,py37-pypi,isort-check,isort-check-examples, isort-check-tests,flake8
+    3.7: py37-min,py37-pypi,flake8
```
Comment (RobbeSneyders) [inline tox.ini:19]: Is there a reason to keep running flake8 on 3.7 instead of 3.9?

File: tox.ini [hunk 2]
```diff
@@ -15,8 +16,9 @@ envlist =
...
     3.8: py38-min
+    3.9: py39-min,isort-check,isort-check-examples,isort-check-tests
```
Comment (RobbeSneyders) [inline tox.ini:21]: We should probably run `py38-pypi` and `py39-pypi` as well. Don't know why this was previously not done for 3.8.

**Review by RobbeSneyders — COMMENTED** [review]
Thx! Left 2 small comments.

File: tox.ini (hunk 1 above)
Comment (cclauss) [reply tox.ini:19]: That old version of flake8 failed on Py39 so I dropped back to Py37 and created a separate flake8 PR.

**Review by RobbeSneyders — APPROVED** [review]
Thx!

**Pull request merged**